src/_capi.pxd

```diff
@@ -319,15 +326,15 @@ cdef extern from "blend2d.h":
 
     ctypedef struct BLTextMetrics:
         pass
-
-    ctypedef void (* BLDestroyImplFunc)(void* impl, void* destroyData)
+        
+    ctypedef void (* BLDestroyExternalDataFunc)(void* impl, void* externalData, void* userData)
     ctypedef BLResult (* BLPathSinkFunc)(BLPathCore* path, const void* info, void* closure)
 
     # BLArray
     BLResult blArrayInit(BLArrayCore* self, uint32_t arrayTypeId)
     BLResult blArrayDestroy(BLArrayCore* self)
     BLResult blArrayReset(BLArrayCore* self)
-    BLResult blArrayCreateFromData(BLArrayCore* self, void* data, size_t size, size_t capacity, uint32_t dataAccessFlags, BLDestroyImplFunc destroyFunc, void* destroyData)
+    BLResult blArrayCreateFromData(BLArrayCore* self, void* data, size_t size, size_t capacity, uint32_t dataAccessFlags, BLDestroyExternalDataFunc destroyFunc, void* destroyData)
```

Comment:
It looks like this function doesn't exist anymore in the [commit](https://github.com/blend2d/blend2d/commit/ca5403c1d02b2bc9d2de581e4cb13e5e80f33860) of blend2d referenced in the submodule file?